packages/react-core/src/components/Badge/Badge.tsx

```diff
@@ -15,14 +17,20 @@ export interface BadgeProps extends React.HTMLProps<HTMLSpanElement> {
 
 export const Badge: React.FunctionComponent<BadgeProps> = ({
   isRead = false,
+  isDisabled = false,
   className = '',
   children = '',
   screenReaderText,
   ...props
 }: BadgeProps) => (
   <span
     {...props}
-    className={css(styles.badge, (isRead ? styles.modifiers.read : styles.modifiers.unread) as any, className)}
+    className={css(
+      styles.badge,
+      (isRead ? styles.modifiers.read : styles.modifiers.unread) as any,
+      (isDisabled ? styles.modifiers.disabled : '') as any,
```

Comment:
since there is no alternative can we say do this instead?
`isDisabled && styles.modifiers.disabled`